src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageVersionsModel.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging.Core;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.UI.Models.Package
+{
+    public class PackageVersionsModel
+    {
+        private INuGetSearchService _nuGetSearchService;
+        private IReadOnlyCollection<VersionInfoContextInfo>? _availableVersions;
+        private bool _dataLoaded;
```

Comment:
`Is-`, like [`Has-`](https://github.com/NuGet/NuGet.Client/pull/6434/files#diff-a3c7d563e397ad982f4475e11ec83fd4639960b73d30a2c700508aaae33db622R30-R31), is a great naming pattern for Booleans (e.g.:  _isDataLoaded).  Makes code even more readable.